boneio/relay/mcp.py

```diff
@@ -24,7 +24,7 @@ def __init__(
     ) -> None:
         """Initialize MCP relay."""
         self._pin: DigitalInOut = mcp.get_pin(pin)
-        self._pin.switch_to_output(value=True)
+        self._pin.switch_to_output(value=False)
```

Comment:
```diff --git a/boneio/relay/mcp.py b/boneio/relay/mcp.py
index 8d7bdaf..c381499 100644
--- a/boneio/relay/mcp.py
+++ b/boneio/relay/mcp.py
@@ -24,11 +24,10 @@ class MCPRelay(BasicRelay):
     ) -> None:
         """Initialize MCP relay."""
         self._pin: DigitalInOut = mcp.get_pin(pin)
-        self._pin.switch_to_output(value=False)
         if output_type == NONE:
             """Just in case to not restore state of covers etc."""
             restored_state = False
-        self._pin.value = restored_state
+        self._pin.switch_to_output(value=restored_state)
         super().__init__(
             **kwargs, output_type=output_type, restored_state=restored_state
```

@pszafer I think it can be even like this ^^ set output state based on restored_state

Reply:
![image](https://user-images.githubusercontent.com/142751/170891709-3cb26c7a-3665-4e28-90f4-2b1a542acc4e.png)

Just tested it on my instance and works as expected.